region/tourian/main/Metroid Room 3.json

```diff
@@ -426,7 +444,11 @@
       "link": [1, 2],
       "name": "Ice Evade",
       "requires": [
-        "Ice"
+        "Ice",
+        {"or": [
+          "canDodgeWhileShooting",
+          {"enemyDamage": {"enemy": "Rinka", "type": "contact", "hits": 1}}
```

Comment:
I feel like morph makes a big difference here, and should maybe be required for lower difficulties. Even on Hard, damageless without morph seems a bit tricky.